### PR TITLE
add crictl config file

### PR DESCRIPTION
**Pull request by mozhuli**
Signed-off-by: mozhuli <21621232@zju.edu.cn>
@heartlock

### CODE REVIEW

**Comment by xlgao-zju** [comment]
@mozhuli 

failure is:
`cmd/crictl/main.go:35:2:warning: exported var RuntimeEndpoint should have comment or be unexported (golint)`

You can run `make lint` before you push.

File: cmd/crictl/main.go [hunk 1]
```diff
@@ -105,7 +120,27 @@ func main() {
 	}
 
 	app.Before = func(context *cli.Context) error {
-		if context.GlobalBool("debug") {
+		configFile := context.GlobalString("conf")
+		config, err := ReadConfig(configFile)
+		if err != nil {
+			return cli.NewExitError("Failed to parse config file", 1)
```
Comment (feiskyer) [inline cmd/crictl/main.go]: Should make this optional. e.g. if no configFile is provided, it should continue to figure out params via flags.

Comment (mozhuli) [inline cmd/crictl/main.go]: updated！

**Comment by mozhuli** [comment]
@xlgao-zju  The priority is: command line arguments > environment variables > profile variables > default variables.

**Comment by feiskyer** [comment]
LGTM. Thanks.

**Pull request merged**